docs/content/3.manual/manual.yml

```diff
@@ -271,17 +271,24 @@ sections:
             input: '"Hello, world!"'
             output: ['"Hello, world!"']
 
-      - title: "`.foo`, `.foo.bar`"
+      - title: "Object Identifier-Index: `.foo`, `.foo.bar`"
```

Comment:
Perhaps `.["string"]` should come before `.string` in the manual.  After all, "how do I get the value of <non-identifier-like string>?" is almost certainly the most FAQ...